.travis.yml

```diff
@@ -45,7 +46,10 @@ before_script:
 
 script:
     - make
-    - if [ -z "$CROSS_COMPILE" ]; then make test; fi
+    - if [ -n "$CROSS_COMPILE" ]; then
+          export EXE_PRE="wine " EXE_EXT=".exe" WINEPREFIX=`pwd`;
```

Comment:
I added another patch to not use EXE_PRE now, to see if binfmt_misc works:
https://travis-ci.org/openssl/openssl/builds/84056196